functions/ocr/app/ocr_test.go

```diff
@@ -0,0 +1,131 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ocr
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	"cloud.google.com/go/storage"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+	"golang.org/x/text/language"
+	pubsubpb "google.golang.org/genproto/googleapis/pubsub/v1"
+)
+
+const (
+	menuName        = "menu.jpg"
+	signName        = "sign.png"
+	bucketName      = "result-bucket-test"
+	imageBucketName = "ocr-image-bucket123"
```

Comment:
Bucket names should be based on the project ID. That way, there is less likely to be an issue with being able to use whatever bucket for running tests. Also, tests run in parallel in multiple projects for golang-samples, so we have to be careful about race conditions. For example, see https://github.com/GoogleCloudPlatform/golang-samples/blob/master/asset/quickstart/export-assets/main.go#L37 (not a test, but the same idea).

Reply:
Done